README.md

```diff
@@ -171,6 +171,14 @@ $.Error :: Type
 Type comprising every Error value, including values of more specific
 constructors such as [`SyntaxError`][2] and [`TypeError`][3].
 
+#### `FiniteNumber`
+
+Type comprising every [`ValidNumber`](#validnumber) value except `Infinity`, `-Infinity`.
```

Comment:
A few minor points:

  - let's wrap lines at 79 characters;
  - `s/,/ and/`; and
  - let's append _(and their object counterparts)_ to this sentence.